core/kazoo_documents/include/kzd_ratedeck.hrl

```diff
@@ -0,0 +1,24 @@
+-ifndef(KZD_RATEDECK_HRL).
+
+-define(MATCH_RATEDECK_DB_ENCODED(RatedeckId)
+       ,<<"ratedeck%2F", RatedeckId/binary>>
+       ).
+-define(MATCH_RATEDECK_DB_encoded(RatedeckId)
+       ,<<"ratedeck%2f", RatedeckId/binary>>
+       ).
+-define(MATCH_RATEDECK_DB_UNENCODED(RatedeckId)
+       ,<<"ratedeck/", RatedeckId/binary>>
+       ).
+
+-define(ENCODE_RATEDECK_DB(RatedeckId)
+       ,<<"ratedeck%2F", RatedeckId/binary>>
+       ).
+-define(encode_RATEDECK_DB(RatedeckId)
+       ,<<"ratedeck%2f", RatedeckId/binary>>
+       ).
+-define(UNENCODED_RATEDECK_DB(RatedeckId)
+       ,<<"ratedeck/", RatedeckId/binary>>
+       ).
```

Comment:
I don't know anything about ratedeck ids nor dbs, but is it at all possible to explicitly match a byte_size here?
It can help a lot (like ?MATCH_ACCOUNT_stuff)